File: dsa/3-arrays/challenges/04-sentence-capitalization/sentence-capitalization.py
def sentence_capitalization(text):
    if not isinstance(text, str):
        raise ValueError("Input must be a string")
    if len(text) == 0:
        raise ValueError("String cannot be empty")

    print(" ".join(word.capitalize() for word in text.lower().split()))


sentence_capitalization("rukito BOnito")

File: dsa/3-arrays/challenges/04-sentence-capitalization/sentenceCapitalization.js
function sentenceCapitalization(text) {
    if (typeof text != "string") throw new Error('Input must be a string')
    if (text.length === 0) throw new Error('String cannot be empty')
    console.log(text.toLowerCase().split(' ').map(word => word[0].toUpperCase() + word.slice(1)).join(' '))
}

sentenceCapitalization('rukito BONITo')